### PR TITLE
[web][mediacontroller][guide] Fix in guides resulting from TV support

### PR DESCRIPTION
**Pull request by rwalczyna**
Information about TV support has been added.
Code examples indentation has been fixed.

Signed-off-by: Rafal Walczyna <r.walczyna@samsung.com>

### CODE REVIEW

File: docs/application/web/guides/multimedia/media-controller.md [hunk 1]
```diff
@@ -596,27 +594,26 @@ You can monitor changes of server abilities using the `addAbilityChangeListener`
...
-  {
-    console.log("displayMode ability changed, server name: " + server.name + ", abilities: ");
-    console.log(JSON.stringify(ability));
-  },
-  onsimpleabilitychanged: function(server, type, ability)
-  {
-    console.log(type + " ability changed, server name: " + server.name + ", ability: " + ability);
-  }
+var listener = {
+    onplaybackabilitychanged: function (server, ability) {
```
[the remark this answers is not in the document]
Comment (rwalczyna) [reply docs/application/web/guides/multimedia/media-controller.md]: Done.

File: docs/application/web/guides/multimedia/media-controller.md [hunk 2]
```diff
@@ -596,27 +594,26 @@ You can monitor changes of server abilities using the `addAbilityChangeListener`
...
+        console.log("playback ability changed, server name: " + server.name +
+            ", abilities: ");
+        console.log(JSON.stringify(ability));
+    },
+    ondisplayrotationabilitychanged: function (server, ability) {
```
[the remark this answers is not in the document]
Comment (rwalczyna) [reply docs/application/web/guides/multimedia/media-controller.md]: Done.

File: docs/application/web/guides/multimedia/media-controller.md [hunk 3]
```diff
@@ -596,27 +594,26 @@ You can monitor changes of server abilities using the `addAbilityChangeListener`
...
+        console.log("display rotation ability changed, server name: " +
+            server.name + ", ability: ");
+        console.log(JSON.stringify(ability));
+    },
+    ondisplaymodeabilitychanged: function (server, ability) {
```
[the remark this answers is not in the document]
Comment (rwalczyna) [reply docs/application/web/guides/multimedia/media-controller.md]: Done.

File: docs/application/web/guides/multimedia/media-controller.md [hunk 4]
```diff
@@ -596,27 +594,26 @@ You can monitor changes of server abilities using the `addAbilityChangeListener`
...
+        console.log("displayMode ability changed, server name: " + server.name +
+            ", abilities: ");
+        console.log(JSON.stringify(ability));
+    },
+    onsimpleabilitychanged: function (server, type, ability) {
```
[the remark this answers is not in the document]
Comment (rwalczyna) [reply docs/application/web/guides/multimedia/media-controller.md]: Done.

File: docs/application/web/guides/multimedia/media-controller.md [hunk 5]
```diff
@@ -537,7 +535,7 @@ Various abilities can be set to give the information to the clients about the su
 
 1. Media controller simple abilities
 
-    Each ability is described by a single [MediaControllerAbilitySupport](../../api/latest/device_api/mobile/tizen/mediacontroller.html#MediaControllerAbilitySupport) value and is not a part of a complex ability structure:
+    Each ability is described by a single value of `MediaControllerAbilitySupport` (in [mobile](../../api/latest/device_api/mobile/tizen/mediacontroller.html#MediaControllerAbilitySupport) , [wearable](../../api/latest/device_api/wearable/tizen/mediacontroller.html#MediaControllerAbilitySupport), and [tv](../../api/latest/device_api/tv/tizen/mediacontroller.html#MediaControllerAbilitySupport) applications)  and is not a part of a complex ability structure:
```
[the remark this answers is not in the document]
Comment (rwalczyna) [reply docs/application/web/guides/multimedia/media-controller.md]: Done.

**Comment by mijin85cho** [comment]
Thanks for the update, @rwalczyna,
I think only https://github.com/Samsung/tizen-docs/pull/865#discussion_r333301550 is required now.
Would you please check line 710 and add version information for TV?

File: docs/application/web/guides/multimedia/media-controller.md [hunk 6]
```diff
@@ -675,29 +674,28 @@ var mcClient = tizen.mediacontroller.getClient();
...
+            new tizen.Bundle({"message" : "Error - Not allowed"}), 13);
+    }
     return new tizen.mediacontroller.RequestReply(
-        new tizen.Bundle({"message": "Error - Not allowed"}), 13);
-  }
-  return new tizen.mediacontroller.RequestReply(new tizen.Bundle({"message": "Success"}), 0);
+        new tizen.Bundle({"message" : "Success"}), 0);
 };
 
 /* Registers to receive display rotation change requests from clients. */
```
[the remark this answers is not in the document]
Comment (rwalczyna) [reply docs/application/web/guides/multimedia/media-controller.md]: Done.

**Pull request merged**